adafruit_esp32spi/adafruit_esp32spi_wsgiserver.py

```diff
@@ -98,9 +98,23 @@ def update_poll(self):
         """
         self.client_available()
         if (self._client_sock and self._client_sock.available()):
+            self.print_remote_ip()
```

Comment:
see note below - line 239 - regarding print statements.

Reply:
I will amend the code.

Reply:
I am not insisting -- just offering an opinion.